### ISSUE TITLE
Suggestion for hyperparameter search grid

### ISSUE THREAD

**Issue by KennethEnevoldsen** [Danish-summarisation/DanSum#10]
w. early stopping on model training.

- learning rate categorical [5e-4, 5e-5, 5e-6],or continuous between [5e-4, 5e-6] 
- dataset density threshold, continuous [1.5-8]
- learning rate scheduler categorical [constant, linear decay, ...]
- warm-up steps [10%]
- fp16: True
gradient clipping, num_beam, min_length, max_length, dropout rate: find suitable values in literature.
